src/credentials/CHIPCert.h

```diff
@@ -274,7 +275,7 @@ struct ChipCertificateData
  */
 struct ValidationContext
 {
-    uint32_t mEffectiveTime;                                 /**< Current time in the CHIP Packed Certificate Time format. */
+    uint32_t mEffectiveTime;                                 /**< Current CHIP Epoch time. */
```

Comment:
```suggestion
    uint32_t mEffectiveTime;                                 /**< Current CHIP Epoch UTC time. */
```
At least assuming that ASN.1 dates/times are UTC, not localtime.  It's not really documented in `ASN1.h`, but `ASN1Reader` seems to fail out if there is no `Z` on the time, so looks like they are?